web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/HentGrunnlagRevurderingRoutes.kt

```diff
@@ -48,4 +53,29 @@ internal fun Route.hentGrunnlagRevurderingRoutes(
             }
         }
     }
+
+    authorize(Brukerrolle.Saksbehandler) {
+        get("$revurderingPath/{vedtakId}/oppsummering") {
```

Comment:
Denne URL-en klæsjer litt siden ressursen er `revurderingPath`; er da litt merkelig at man kan sende inn vedtakId. En annen ting er at den heter "oppsummering", men returnerer grunnlagsdata og vilkårsvurderinger.
Hva med noe ala:
```suggestion
        get("$revurderingPath/historisk/vedtak/{vedtakId}/grunnlagsdataOgVilkårsvurderinger") {
```
?

Reply:
Sounds good! 👍